src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml

```diff
@@ -44,15 +44,13 @@
         <service id="sylius.promotion_action.fixed_discount" class="Sylius\Component\Core\Promotion\Action\FixedDiscountPromotionActionCommand">
             <argument type="service" id="sylius.proportional_integer_distributor" />
             <argument type="service" id="sylius.promotion.units_promotion_adjustments_applicator" />
-            <argument type="service" id="sylius.currency_converter" />
             <tag name="sylius.promotion_action" type="order_fixed_discount" label="Order fixed discount" />
         </service>
         <service id="sylius.promotion_action.unit_fixed_discount" class="Sylius\Component\Core\Promotion\Action\UnitFixedDiscountPromotionActionCommand">
             <argument type="service" id="sylius.factory.adjustment" />
             <argument type="service" id="sylius.promotion_filter.price_range" />
             <argument type="service" id="sylius.promotion_filter.taxon" />
             <argument type="service" id="sylius.promotion_filter.product" />
-            <argument type="service" id="sylius.currency_converter" />
```

Comment:
The promotions will be reworked separately, do the minimum you need to do to make scenarios for single channel green. Scenarios for multi currency and multi channel can be marked temporarly `@todo`